pkgs/applications/networking/cluster/helm/default.nix

```diff
@@ -2,14 +2,14 @@
 
 buildGoModule rec {
   pname = "kubernetes-helm";
-  version = "3.9.1";
+  version = "3.9.2";
   gitCommit = "a7c043acb5ff905c261cfdc923a35776ba5e66e4";
```

Comment:
git commit needs to be updated

Reply:
removed in 623192327f8e0652acfca3f083164eab779d44f7